Src/Plugins/Security/SEC101_052.FacebookAccessTokenValidator.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class FacebookAccessTokenValidator : ValidatorBase
```

Comment:
>FacebookAccessTokenValidator [](start = 17, length = 28)

filename should be 014

Reply:
Done

---
In reply to: [586810142](https://github.com/microsoft/sarif-pattern-matcher/pull/255#discussion_r586810142) [](ancestors = 586810142)